patches/minecraft/net/minecraft/world/entity/player/Player.java.patch

```diff
@@ -200,6 +200,15 @@
        if (p_36347_.m_6097_()) {
           if (!p_36347_.m_7313_(this)) {
              float f = (float)this.m_21133_(Attributes.f_22281_);
+@@ -1037,7 +_,7 @@
+             } else {
+                f1 = EnchantmentHelper.m_44833_(this.m_21205_(), MobType.f_21640_);
+             }
+-
```

Comment:
You can just add an extra line here instead of replacing to reduce the patch size to one line instead of two.